live-examples/js-examples/array/array-iterator.js

```diff
@@ -0,0 +1,10 @@
+const array1 = ['a', 'b', 'c'];
+const iterator = array1[Symbol.iterator]();
```

Comment:
It's a convention to use a numerical suffix when a variable name could be confused with an identifier: https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#variable-naming.